.gitlab-ci.yml

```diff
@@ -141,6 +140,7 @@ deploy:sls:branch_remove:
   environment:
     name: review/$CI_EXTERNAL_PULL_REQUEST_IID
     action: stop
+    auto_stop_in: 3 days
```

Comment:
This is an experiment more than anything else. The previous `auto_stop_in` wasn't working.